lib/dockerregistry/inventory.go

```diff
@@ -208,9 +223,15 @@ func (m *Manifest) finalize() error {
 // (there can only be 1 source registry).
 func ParseManifestsFromDir(
```

Comment:
https://golang.org/pkg/path/filepath/#Walk already only walks recursively inside the given directory and does not follow symlinks, so changing this function is unnecessary.